doc/api/next_api_changes/behavior/20405-JMK.rst

```diff
@@ -0,0 +1,8 @@
+Make norm from scale now public
+===============================
+
+Downstream libraries can take advantage of ``make_norm_from_scale`` in 
+``colors.py`` to create a `~.colors.Normalize` subclass directly from 
+an existing scale.  Usually norms have a scale, and the advantage of 
+having a  `~.scale.ScaleBase` attached to a norm is to provide a 
+default scale for the colorbar.  
```

Comment:
perhaps be more explicit: "provide a scale, and associated tick locators and formatters, for the colorbar"